inscriptions/ISic003870.xml

```diff
@@ -182,17 +190,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI334309">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SS">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Καρχηδ-<lb n="2"/>ονί<supplied reason="omitted">ου</supplied>.
-                    <lb n="3"/>
+                    <lb n="1"/>Καρχηδ
+                    <lb n="2" break="no"/>ονί<supplied reason="omitted">ου</supplied>
```

Comment:
not omitted but corrected, so choice with sic and corr, and so the apparatus in turn is not required.  The stone reads: ΟΝΙΥΟ, when it should be corrected to ΟΝΙΟΥ.